dataflow/src/main/java/org/checkerframework/dataflow/cfg/block/Block.java

```diff
@@ -41,4 +42,11 @@
      * @return the predecessors of this basic block
      */
     Set<Block> getPredecessors();
+
+    /**
+     * Returns all the successors of this basic block.
+     *
+     * @return all the successors of this basic block
+     */
+    Collection<Block> getSuccessors();
```

Comment:
Should this be `Set` like for `getPredecessors()`? We don't need duplicates in the result.

Reply:
This forces use of `LinkedHashSet` throughout instead of the more efficient `ArrayList`.  I don't see anywhere that clients of `getPredecessors` require (for correctness) that the result is a set.